sdk/entra/functions-authentication-events/review/functions-authentication-events.api.md

```diff
@@ -0,0 +1,144 @@
+## API Report File for "@azure/functions-authentication-events"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export interface ActionableCloudEventResponse<TEventAction extends AuthEventAction> extends ActionableResponse<TEventAction> {
+    // (undocumented)
+    oDataType: string;
+}
+
+// @public
+export interface ActionableResponse<TEventAction extends AuthEventAction> extends AuthEventResponse {
+    actions: TEventAction[];
+}
+
+// @public
+export interface AuthenticationEventContext {
+    authenticationProtocol: "OAUTH2.0" | "SAML" | "WS-FED" | "unknownFutureValue" | "";
```

Comment:
What does the `""` mean? Would it be possible to have undefined instead? `authenticationProtocol?: "OAUTH2.0" | "SAML" | "WS-FED" | "unknownFutureValue" `

Reply:
Done